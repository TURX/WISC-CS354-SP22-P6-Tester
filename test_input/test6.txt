0 10000 1000