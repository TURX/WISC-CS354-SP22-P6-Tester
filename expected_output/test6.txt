Reading character starting at index 0 to 10000 with an interval of 1000

Reading character at index 0   : data = A : hit count = 0   : miss count = 1   : read data count = 1
Reading character at index 1000 : data = e : hit count = 0   : miss count = 2   : read data count = 2
Reading character at index 2000 : data =   : hit count = 0   : miss count = 3   : read data count = 3
Reading character at index 3000 : data = ' : hit count = 0   : miss count = 4   : read data count = 4
Reading character at index 4000 : data = o : hit count = 0   : miss count = 5   : read data count = 5
Reading character at index 5000 : data = e : hit count = 0   : miss count = 6   : read data count = 6
Reading character at index 6000 : data = a : hit count = 0   : miss count = 7   : read data count = 7
Reading character at index 7000 : data = t : hit count = 0   : miss count = 8   : read data count = 8
Reading character at index 8000 : data = w : hit count = 0   : miss count = 9   : read data count = 9
Reading character at index 9000 : data = i : hit count = 0   : miss count = 10  : read data count = 10
